sqlmesh/core/model/definition.py

```diff
@@ -2440,3 +2448,6 @@ def _meta_renderer(
 def get_model_name(path: Path) -> str:
     path_parts = list(path.parts[path.parts.index("models") + 1 : -1]) + [path.stem]
     return ".".join(path_parts[-3:])
+
+
+NO_MICROSECONDS = {"clickhouse": (exp.DataType.Type.DATETIME, exp.DataType.Type.TIMESTAMP)}
```

Comment:
I'm sorry, I think I didn't express myself clearly. I meant
```
TIME_COLUMN_NO_MICROSECONDS_TYPES
```
suggesting to replace the `FRACTION_SECONDS` part only. Apologies about the confusion